appcues/src/main/java/com/appcues/debugger/ui/DebuggerPanel.kt

```diff
@@ -66,7 +66,7 @@ internal fun BoxScope.DebuggerPanel(debuggerState: MutableDebuggerState, debugge
         Spacer(
             modifier = Modifier
                 .fillMaxSize()
-                .background(AppcuesColors.DebuggerBackdrop)
+                .background(Color(color = 0x51000000))
```

Comment:
should this be using a theme color?

Reply:
I had it in there but I think this will never differ from dark to light mode or any other theme so decided to just hardcode since its black with some transparency